core/lib/generators/solidus/install/install_generator.rb

```diff
@@ -143,11 +143,6 @@ def configure_application
       end
     end
 
-    def plugin_install_preparation
```

Comment:
Are you going to need all of the old plugin installation system for https://github.com/solidusio/solidus/pull/4673?

Reply:
Being #4673 targeted at 3.2 I don't think we need to keep this old system in place, and I don't see a scenario in which we'll keep needing it… but even in that case it should be easy to add it back.

Reply:
Oh, yeah, sorry, forget about it.